Gemfile

```diff
@@ -7,6 +7,7 @@ gemspec
 gem 'rake', :require => false
 
 group :test do
+  gem 'faraday', '0.15.4'
```

Comment:
You probably want to bump this over in `vmfloaty.gemspec`, not here

Reply:
Fixed per slack convo, deleting reference to Faraday in spec headers.